src/main/java/app/linguistai/bmvp/configs/XPConfiguration.java

```diff
@@ -26,11 +30,27 @@ public Map<String, Integer> getLevels() {
         return levels;
     }
 
+    public Long getBaseLevel() {
+        return baselevel;
+    }
+
+    public Long getLevelCoefficient() {
+        return levelcoefficient;
+    }
+
     public void setValues(Map<String, Integer> values) {
         this.values = values;
     }
 
     public void setLevels(Map<String, Integer> levels) {
         this.levels = levels;
     }
+
+    public void setBaseLevel(Long baselevel) {
```

Comment:
maybe you can use lombok instead of writing these methods

Reply:
The names of the "variables" that are set in application.properties (which this class reads and loads) are case-sensitive, therefore the lombok generated codes would be setBaselevel (noncomplient for method naming), hence the manual definitions. we can change this once/if we change xp configuration loader.